pynecone/components/overlay/popover.py

```diff
@@ -108,6 +108,9 @@ def create(
         Returns:
             The popover component.
         """
+        if props.get("trigger_comp") == "click" or props.get("trigger_comp") == "hover":
```

Comment:
Sorry I actually meant we should change line 90 (and maybe the following ones also) to be `trigger_comp`, `header_comp`, etc. So it's clear that those are components that are passed in, while the normal `trigger` prop should remain untouched. Then we don't need this special logic here to check for "click" or "hover". Does that make sense?

Reply:
oh, i konw your mind ~ i will fix it now